docs/api/materials/ShaderMaterial.html

```diff
@@ -44,6 +44,18 @@ <h1>[name]</h1>
 				If you don't want the [page:WebGLProgram] to add anything to your shader code, you can use
 				[page:RawShaderMaterial] instead of this class.
 			</li>
+			<li>
+				You can use the directive #pragma unroll_loop in order to unroll a *for* loop in GLSL by the shader preprocessor.
+				The directive has to be placed right above the loop. The loop itself has to correspond to a defined standard.
+				<ul>
+					<li>
+						The loop has to be normalized/well-behaved.
```

Comment:
>The loop has to be normalized/well-behaved.

You probably should explain those terms.

Perhaps it would also be a good idea to add an example code snippet.

Reply:
👍